### PR TITLE
Improve formatting

### PR DESCRIPTION
**Pull request by nirs**
- Reformat long optins idiomatically
- Unify whitespace

### CODE REVIEW

**Review by jandubois — DISMISSED** [review]
Thanks, LGTM

Now the `enum` a little higher up is the only place that still uses 4 spaces indentation; maybe make everything consistent?

**Comment by nirs** [comment]
> Thanks, LGTM
> 
> Now the `enum` a little higher up is the only place that still uses 4 spaces indentation; maybe make everything consistent?

I did not notice, adding another commit.

How about adding a clang-format file to auto format the code?

And change to more common and readable style, like linux kernel style? It is hard to work with 2 space indent, and the code look too dense.

**Comment by nirs** [comment]
@jandubois enum fixed now.

File: cli.c [hunk 1]
```diff
@@ -82,22 +82,18 @@ struct cli_options *cli_options_parse(int argc, char *argv[]) {
...
+    {"vmnet-interface",     required_argument,  NULL,   CLI_OPT_VMNET_INTERFACE},
+    {"vmnet-gateway",       required_argument,  NULL,   CLI_OPT_VMNET_GATEWAY},
+    {"vmnet-dhcp-end",      required_argument,  NULL,   CLI_OPT_VMNET_DHCP_END},
+    {"vmnet-mask",          required_argument,  NULL,   CLI_OPT_VMNET_MASK},
+    {"vmnet-interface-id",  required_argument,  NULL,   CLI_OPT_VMNET_INTERFACE_ID},
+    {"vmnet-nat66-prefix",  required_argument,  NULL,   CLI_OPT_VMNET_NAT66_PREFIX},
+    {"pidfile",             required_argument,  NULL,   'p'},
+    {"help",                no_argument,        NULL,   'h'},
+    {"version",             no_argument,        NULL,   'v'},
+    {0, 0, 0, 0},
```
Comment (AkihiroSuda) [inline cli.c:96]: We should use `clang-format` or something for maintaining the format.

Does `clang-format` or something support keeping this whitespacing?

Comment (nirs) [reply cli.c:96]: No idea but even if not having automatic formatting is better.

Comment (nirs) [reply cli.c:96]: All the default styles (LLVM, GNU, Google, Chromium, Microsoft, Mozilla, WebKit) destroy the nice table. Maybe there is an option to align tables, or to disable formatting of part of a file.

Comment (jandubois) [reply cli.c:96]: I haven't tested it, but according to [Disabling Formatting on a Piece of Code](https://clang.llvm.org/docs/ClangFormatStyleOptions.html#disabling-formatting-on-a-piece-of-code) you can disable it for a block like this:

```c
// clang-format off
  const struct option longopts[] = {
    {"socket-group",        required_argument,  NULL,   CLI_OPT_SOCKET_GROUP},
    {"vmnet-mode",          required_argument,  NULL,   CLI_OPT_VMNET_MODE},
    {"vmnet-interface",     required_argument,  NULL,   CLI_OPT_VMNET_INTERFACE},
    {"vmnet-gateway",       required_argument,  NULL,   CLI_OPT_VMNET_GATEWAY},
    {"vmnet-dhcp-end",      required_argument,  NULL,   CLI_OPT_VMNET_DHCP_END},
    {"vmnet-mask",          required_argument,  NULL,   CLI_OPT_VMNET_MASK},
    {"vmnet-interface-id",  required_argument,  NULL,   CLI_OPT_VMNET_INTERFACE_ID},
    {"vmnet-nat66-prefix",  required_argument,  NULL,   CLI_OPT_VMNET_NAT66_PREFIX},
    {"pidfile",             required_argument,  NULL,   'p'},
    {"help",                no_argument,        NULL,   'h'},
    {"version",             no_argument,        NULL,   'v'},
    {0, 0, 0, 0},
  };
// clang-format on
```

Comment (nirs) [reply cli.c:96]: I opened #104

**Review by jandubois — APPROVED** [review]
Thanks, LGTM

**Pull request merged**